### DISCUSSION TITLE
How to change toggle on to bg-blue-700?

### DISCUSSION THREAD

**Discussion by crapthings** [saadeghi/daisyui#729] [Q&A] [answered]
```html
<input type="checkbox" class="toggle bg-blue-700" checked>
```

![image](https://user-images.githubusercontent.com/1147704/163822257-4ae8617c-0aed-49ed-94c0-96deebcd7209.png)

![image](https://user-images.githubusercontent.com/1147704/163822264-562244b6-d415-4b9c-afbf-092533a27154.png)

```html
<input type="checkbox" class="toggle" checked>
```

i want to keep off original gray color

![image](https://user-images.githubusercontent.com/1147704/163822358-61c8b0c9-7e23-498c-9e3e-c8ec2de10f13.png)

**Comment by saadeghi** [comment, marked as answer]
You can do it like this:
```html
<div class="m-10">
  <input
    checked
    type="checkbox"
    class="toggle checked:bg-blue-700 checked:border-blue-700"
  >
</div>
```
https://play.tailwindcss.com/OF9d6Kwbbq
